owncloud/owncloud.py

```diff
@@ -703,6 +703,26 @@ def mkdir(self, path):
             path += '/'
         return self._make_dav_request('MKCOL', path)
 
+    def makedirs(self, exist_ok=False):
```

Comment:
should be `def makedirs(self, path, exist_ok=False`

Reply:
Yes, I definitely missed it. Thanks!